README.md

```diff
@@ -67,6 +67,16 @@ Once the model is created, all available Swagger API operations can be accessed
     });
   ```
 
+Generated methods use swagger's __operationId__.  If there is a nested URL path, the connector will append an `_` in place of the `/`.  For example, looking at a swagger.json file, you would see something like:
+
+```
+paths: {
+  /weather/forecast: {
+    operationId: "weather.forecast",
+```
+
+This means you can call `model.weather_forecast()` to access this endpoint programmatically.
```

Comment:
@rblalock : Hi Rick, real picture here is:
 The connector is using a module: [swagger-client](https://github.com/swagger-api/swagger-js) which decides on how to name a method as follows and passes it to connector.
For the case:
```
/weather/forecast:
  get:
    ...
    operationId: weather.forecast
    ...
```
when **operationId** is available, it takes precedence
   - if `operationId` is of format equivalent to calling a nested function such as: `weather.forecast` --> the resulting method name will replace `.` with `_` i.e. 
 `weather.forecast` ---> `weather_forecast()`

When `operationId` is missing, the method name is constructed as:
 `<operationType (i.e. get, post, etc)>` + `_` + `<path parts seperated by underscore>`

For the example below: 
```
/weather/forecast:
  get:
    ...
```
the resulting method name will be: `get_weather_forecast`.
Hope that helps.

Could you please reword the example accordingly and resolve the merge conflict. Thanks.